app/router.js

```diff
@@ -8,5 +8,7 @@ export default class Router extends EmberRouter {
 
 Router.map(function () {
   this.route('login', { path: '/' });
+  this.route('forgot-password', { path: '/forgotpassword' });
```

Comment:
Why to have separate path defined?

Reply:
they does not gave any forgorpassword in login so i added i asked charitha and rahul for that after only they said to add